src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -816,6 +816,22 @@ public static void EndsWith(string s, string value, StringComparison comparisonT
         }
 
         [Theory]
+        [InlineData("Hello", 'o', true)]
+        [InlineData("Hello", 'O', false)]
+        [InlineData("o", 'o', true)]
+        [InlineData("o", 'O', true)]
```

Comment:
Oops, it looks like they ended up wanting to match .IndexOf(char) behavior. That is still case sensitive -it's ordinal.

https://msdn.microsoft.com/en-us/library/kwb0bwyd(v=vs.110).aspx